packages/dataviz/src/components/LineChart/LineChart.stories.tsx

```diff
@@ -0,0 +1,317 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import  LineChart, { LineChartProps } from './LineChart.component';
+
+const Template: Story<LineChartProps> = args => <LineChart {...args} />;
```

Comment:
```suggestion
```

you can use the new CSF syntax:
```suggestion
export const FullyCustomisedLineChart = Template.bind({});```
```suggestion
const Template: Story<LineChartProps> = args => <LineChart {...args} />;
```

```
export const FullyCustomisedLineChart = {
   args: {...}
};
```